.github/workflows/backport-trigger.yml

```
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+name: Backport comment trigger
+
+on: issue_comment
+
+jobs:
+  backport:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/backport')
+    steps:
+      - name: Extract Command
+        id: command
+        uses: xt0rted/slash-command-action@bf51f8f5f4ea3d58abc7eca58f77104182b23e88 # v2.0.0
+        with:
+          command: backport
```

If you didn't put a token intentionally or you tested, ignore this comment (and below comments about token). Otherwise we may be need to use token here to interact with GitHub API?

```suggestion
        with:
          command: backport
          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

---

The latest version of the backport action doesn't require explicitly passing in the repo token